.github/workflows/nightly.yml

```diff
@@ -6,6 +6,14 @@ on:
       - cron: '0 0 * * *'
 
     workflow_dispatch:
+      inputs:
+        push_benchmark_results_to_gh_pages:
+          description: "When set to true, the workflow pushes all benchmarking results to gh-pages UI "
+          type: choice
+          options:
+              - 'true'
+              - 'false'
+          default: 'false'
```

Comment:
This option is to force-push results to the github branch where the benchmark results are maintained for UI